src/main/java/com/ericsson/gerrit/plugins/eiffel/handlers/DataBaseHandler.java

```diff
@@ -91,12 +73,16 @@ public String getEventID(final Table table, final String keyValue) throws Connec
             LOGGER.error("Error when trying to fetch values from database: {}\n{}", e.getMessage(), e);
         }
 
+        if (eventID.isEmpty()) {
+            throw new RuntimeException("Database did not return any value for this query");
```

Comment:
Maybe NoSuchElementException is a better exception? :)

Reply:
done